2018-02-24-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,74 @@
+- Beatmap submissions may contain files up to 15 MB.
+- Storyboarding will not be considered in the judging process.
+- Custom hitsounds **are** allowed to be used.
+- You may submit **one** difficulty for **one** song. Multiple submissions for multiple songs will not be accepted and will result in all submissions to be voided.
+- Maps submitted to the contest must not be uploaded with the beatmap submission system (BSS) until the contest has concluded and the winners have been announced.
+
+## Judging Criteria
+
+All entries will be judged by the following criteria:
+
+- Musical Relevance (10P) - How well does the entry follow the music with its choice of rhythms, emphasis, spacing, etc?
+- Creativity (10P) - How unique and/or innovative is the entry in its use of concepts and design?
+- Technique (10P) - How well does the entry utilize the editor and its tools to create structure, patterning, etc? 
+- Impression (10P) - How does the judge view the map holistically (as the entire creation)? Do all of the concepts used in the map work well together and are they executed in a consistent and polished manner?
+- Hitsounding (5P) - How well does the entry utilize hitsounding to emphasize and follow the music?
+
+**We are looking for capable judges! If you are interested, please post in the [Judging Call thread](https://osu.ppy.sh/community/forums/topics/691326) or send a message to [Chaos](https://osu.ppy.sh/u/2628870). Please be aware that we can only accept people who have the resources (knowledge, time, reliability) to properly judge the entries.**
+
+## Prizes
+
+After the entries have been judged and the results have been finalized, each of the top three maps for each track will receive the following prizes:
+
+- **1st place**: 3 months of osu!supporter, 6 Olympiad Points (Results in 'Mapping Olympian' Badge)
+- **2nd place**: 2 months of osu!supporter, 4 Olympiad Points
+- **3rd place**: 1 months of osu!supporter, 2 Olympiad Points
+
+For Olympiad contests, we will be using a point system to award the 'Mapping Olympian' badge. This means that top placing results across multiple contests will grant you the opportunity to be awarded a badge even if you do not win a contest. Every 6 points earned will grant you a level of the badge, with incrementation on the badge for each level achieved. The expected baseline for points rewards will be 6,4,2 for the top 3 entries respectively but the specific rewards will vary per contest based on expected effort and challenge for the participants.
+
+Some of you may be wonder where the *Elite Mapper* title for this contest. We considered a few options but none provided an adequate balance between fairness, demonstrating mapping prowess, and integrity for this particular contest. We will have contests in the future where the title will be rewarded.
```

Comment:
first sentence here has some grammar issues

probably "Some of you may be wondering why there won't be an *Elite Mapper* title rewarded for this contest."